_posts/2021-12-08-naip-2021-acquisition-complete.md

```diff
@@ -0,0 +1,13 @@
+---
+title: '2021 Aerial Photography Acquisition Completed'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2021-12-08 12:00:00
+categories:
+  - Featured
```

Comment:
**question**: do we want this to be featured with such little content?

Reply:
I think so- this is a long-awaited product that we've been fielding questions about for at least 6 months.

Reply:
I think so too